ds-card-reader/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/edgexfoundry/device-sdk-go/v2 v2.2.0
 	github.com/edgexfoundry/go-mod-core-contracts/v2 v2.2.0
 	github.com/gvalkov/golang-evdev v0.0.0-20180516222720-b6f418b1fe5a
-	github.com/intel-iot-devkit/automated-checkout-utilities v1.0.0
+	github.com/pkg/errors v0.9.1
```

Comment:
So the use of this package changes the BOM for this RI. We have avoid it in KP as it isn't supported well and other issues. IMO we shouldn't use it here either.

Reply:
should I remove github.com/pkg/errors v0.9.1 and put github.com/intel-iot-devkit/automated-checkout-utilities v1.0.0 back?

Reply:
No, errors is pulled in due to the use of errors.Wrap(). Avoid using this function.

Reply:
changed all errors.Wrap() to fmt.errorf ()